docs/achtergronddocumentatie/artefacten/0796.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page-with-side-nav
+title: F0796 - Wijzig klantcontact
+date: 20-12-2023
+---
+
+# F0796 - Wijzig klantcontact
+
+## Functionaliteit
+
+De functie wijzigt gegevens van een bestaand `klantcontact`. 
+
+- Het ID en het nummer zijn niet te wijzigen, want dit zijn uniek identificerende gegevens. 
+
+Vamuit functioneel perspectief is een klantcontact dat eenmaal is afgelopen niet meer wijzigbaar. 
+Daar zijn 2 uitzeonderingen op. 
+- Het doorvoeren van correcties. Dit is functtioneel iets anders dan een wijziging en zal ook in een aparte functie worden beschreven.
+- Het wijzigen van de inhoud als tijdens een klantcontact de klant wordt doorverbonden naar een andere medewerker. In de beleving van de klant is dit één klantcontact.
+ 
+Alleen het wijzigen van de inhoud wordt hier technisch beschreven. 
+
+Eerst wordt het te wijzigen klantcontact opgehaald met [F3896 - Zoek klantcontact](./3896) of [F8618 - Vraag klantcontact op](./8618)
+
+Vervolgens wordt de wijziging doorgevoerd.
+
+### API Call 1
+
+- patch ../partij/e8203b1b-f9e1-4fa1-96ce-2bcf36e061c2
+
+### Requestbody
+
+In de requestbody van de patch worden alleen de property *inhoud* opgenomen. 
+De opgehaalde inhoudtekst wordt aangevuld en niet vervangen.  
```

Comment:
Ik weet niet of ik helemaal begrijp wat "aanvullen in plaats van vervangen" hier betekent. Je vervangt toch altijd de hele waarde van een property?

In deze casus is het misschien aanvullend illustratief om de response van het initiële opzoeken of opvragen van het klantcontact te tonen om te laten zien wat er gewijzigd wordt.